### ISSUE TITLE
How to use multiple image? 

### ISSUE THREAD

**Issue by franksuni** [chaojie/ComfyUI-dust3r#3]
Say if I'm going to generate a video from SV3D, how can I use the SV3D result directly into dust3r? I keep getting errors when I try to use 3 or more images. Is there a way to bypass multiple image issues? or is there a way that I can use the SV3D result directly?

**Comment by jtydhr88** [comment]
Yes, it can deal with multiple images, but not here(Comfyui custom node), I tested the original dust3d source and it works well on multiple images(such as generated images from SV3D), I believe this repo is out-of-date, and I recommend you try the original dust3d

**Comment by jtydhr88** [comment]
I may help to update the repo but I am not sure the author of this repo still maintains it or not, you can find I have opened another PR but still be there, not merge yet

**Comment by DwanZhang-AI** [comment]
Hi, it seems that you r using dust3r for sv3d, is there any result?
